framework/wazuh/core/engine/content.py

```diff
@@ -43,6 +43,26 @@ async def get_resources(self, type: ResourceType, id_: str, policy_type: PolicyT
             resource = self._db[key]
         return {"status": "OK", "error": None, "content": resource.to_dict()}
 
+    async def get_multiple_resources(self, type: ResourceType, names: list[str], policy_type: PolicyType) -> dict:
+        """
+        Retrieve multiple content resources by their names.
+
+        Parameters
+        ----------
+        type : ResourceType
+            The type of the resource.
+        names : list[str]
+            List of resource names to retrieve.
+        policy_type : PolicyType
+            The policy type associated with the resources.
+
+        Returns
+        -------
+        dict
+            Dictionary with the status, error, and a list of found resources.
+        """
+        return {"status": "OK", "error": None, "content": []}
+
```

Comment:
This must return the elements from the provisional in-memory storage. But using the id instead of the name